### ISSUE TITLE
Save tags action duplicates the current tags 

### ISSUE THREAD

**Issue by elainekamlley** [LibraryOfCongress/concordia#806]
**What behavior did you observe? Please describe the bug**
When I add a new tag to a group of existing tags, when I hit save tags. the existing tags are duplicated

**How can we reproduce the bug?**

1. Go to a page that already has tacks 
2. Add a new tag 
3. Save tags 
4. Refresh page
5. You should see duplicates of the existing tags along with the new tag

**What is the expected behavior?**
To see the tag I added along with the existing tags without the dupes

GIF to show actions taken
![tagbug 2019-02-08 13_25_37](https://user-images.githubusercontent.com/7362915/52497838-201ed300-2ba5-11e9-93df-055da45b25cf.gif)

**Desktop (please complete the following information):**

-   iOS
-   Browser: chrome

**Comment by acdha** [comment]
I can't reproduce this

**Comment by acdha** [comment]
Ah, I think the instructions above are irrelevant. What matters is that the first set of tags have to have been entered by someone else. Tags are stored as (user, tag) pairs. When you save the list of tags, you don't have permission to edit someone else's tag list and so the values are repeated for your account as well. We should make the display only display the unique values and, perhaps, a count.

**Comment by acdha** [comment]
See #807 — this is technically working correctly in the sense that tag lists are per-user but the display is global but it's certainly not a great user experience and it's raising again on the questions about what tagging is intended to do.

**Comment by elainekamlley** [comment]
Even if the instructions are irrelevant to what is actually happening, that was how I experienced it. I came to a page that had tags already on it. Added one tag then saved and reloaded the page. I then saw duplicates of the tags that were already on the page along with my new one. I did not have the insight of how it was built. Glad there is a solution for now.

As we discussed as a team, we will come back to what tagging is meant to do in the near future. In the meantime, global display will prevent some confusion for your users. Thanks for jumping in.
